test/README.md

```diff
@@ -0,0 +1,77 @@
+# Tests
+
+This folder contains automated tests for this module. All of the tests are written in [Go](https://golang.org/). Most of these are "integration tests" that deploy real infrastructure using Terraform and verify that infrastructure works as expected using a helper library called [Terratest](https://github.com/gruntwork-io/terratest).
+
+## WARNING WARNING WARNING
+
+**Note #1**: Many of these tests create real resources in an AWS account and then try to clean those resources up at the end of a test run. That means these tests may cost you money to run! When adding tests, please be considerate of the resources you create and take extra care to clean everything up when you're done!
+
+**Note #2**: Never forcefully shut the tests down (e.g. by hitting `CTRL + C`) or the cleanup tasks won't run!
+
+**Note #3**: We set `-timeout 60m` on all tests not because they necessarily take that long, but because Go has a default test timeout of 10 minutes, after which it forcefully kills the tests with a `SIGQUIT`, preventing the cleanup tasks from running. Therefore, we set an overlying long timeout to make sure all tests have enough time to finish and clean up.
+
+## Running the tests
```

Comment:
Can you add note that the `to-test` label needs to be set by a repo member before the test will run?